ui/app/components/auth/splash-page.hbs

```diff
@@ -0,0 +1,119 @@
+{{!
```

Comment:
I wasn't really sure what to name this to distinguish it from `auth/page` (now renamed to `auth/form-page`). I had considered `auth/route-page` or `auth/cluster-page` but `-splash` seemed more descriptive?

Reply:
I completely agree, I was really struggling with naming 🙈. Since the pattern so far has been routes directly corresponding to the page components what do you think about this? (Your suggestions helped get the naming part of my brain going!) 
 `auth/form-page` -> `auth/login-form`
`auth/splash-page` -> `auth/page` ?